includes/CMB2_Base.php

```diff
@@ -379,6 +379,43 @@ public static function maybe_hook( $val, $hook_name, $hook_function ) {
 	}
 
 	/**
+	 * Arguments for custom wp_kses sanitization/validation calls.
+	 *
+	 * @since  2.2.4
+	 * @param  string $type The field type being escaped.
+	 * @return array        An array of wp_kses HTML parameters allowed.
+	 */
+	public function kses_args( $type ) {
+		$params = array();
+
+		switch ( $type ) {
+			case 'group_wrap_attributes':
+				$params = array(
+					'class' => array(),
+					'style' => array(),
+				);
+				break;
+			case 'remove_disabled':
+				$params = array(
+					'disabled' => array(),
+				);
+		}
+
+		return $params;
+	}
+
+	/**
+	 * Helper function that returns whether a field group is repeatable or not.
+	 *
+	 * @since  2.2.4
+	 * @param  object $field_group The field group.
+	 * @return boolean
+	 */
+	public function is_repeatable( $field_group ) {
```

Comment:
the only way this feels better to me is if this is moved to `CMB2_Field`, and even then, I could see creating an `CMB2_Field::is( $param )` method that allows checking boolean field parameters (so `$field_group->args( 'repeatable' );` would become `$field_group->is( 'repeatable' );`).